.github/workflows/reusable_build_pure_python.yml

```diff
@@ -0,0 +1,38 @@
+name: reusable_build_pure_python
+
+on:
+  workflow_dispatch:
+    inputs:
+      python_folder:
+        description: "Location of the py interface folder"
+        required: true
+        type: string
+  workflow_call:
+    inputs:
+      python_folder:
+        description: "Location of the py interface folder"
+        required: true
+        type: string
+
+jobs:
+
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v4
+      with:
+        python-version: "3.x"
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install setuptools wheel twine
+    - name: Build and publish
+      env:
+        TWINE_USERNAME: __token__
+        TWINE_PASSWORD: ${{ secrets.PYPI_API_TOKEN }}
+      run: |
+        cd ${{inputs.python_folder}}
+        python setup.py sdist bdist_wheel --universal
```

Comment:
We should change from setup.py to using pyproject.toml. That will change the build command for wheels.